app/models/hold.rb

```diff
@@ -104,6 +138,10 @@ def status_code
     fields['status']
   end
 
+  def bib_summary
+    "#{title} (#{call_number})"
```

Comment:
good idea 👍 I should do the same for checkout

Reply:
might be better to send over to bib_record model

Reply:
yes!